features/270-next-skycli/commands.md

```diff
@@ -66,13 +66,20 @@ View the current config.
 #### Example
 ```
 $ skycli config view
-[cluster]
-email=user@skygear.io
-cluster-server=https://1.2.3.4
-admin-key=
-
-[app]
-app=myapp
+api_version: v1
+clusters:
+  - name: skygeario
+    cluster:
+      env: cloud
+      endpoint: 'https://controller.example.com'
+      api_key: api-key
+users: []
```

Comment:
For completeness, either include the user `skygeario` here or omit `user` in the context `skygeario`.

Reply:
This output is possible, if there is no user associated with the `skygeario` context. In that case, the context would refer to an nonexistent user and `auth login` would create/update it.